src/edu/csus/ecs/pc2/imports/ccs/ContestSnakeYAMLLoader.java

```diff
@@ -3288,12 +3351,13 @@ private boolean loadCCSTSVFiles(IInternalContest contest, File cdpConfigDirector
         boolean loaded = false;
 
         String teamsTSVFile = cdpConfigDirectory.getAbsolutePath() + File.separator + LoadICPCTSVData.TEAMS_FILENAME;
+        String teamsTSV2File = cdpConfigDirectory.getAbsolutePath() + File.separator + LoadICPCTSVData.TEAMS2_TSV;
 
         String groupsTSVFile = cdpConfigDirectory.getAbsolutePath() + File.separator + LoadICPCTSVData.GROUPS_FILENAME;
 
         // only load if both tsv files are present.
```

Comment:
The above comment should be updated to reflect the new code below.  It should probably say something like `only load if both a teams tsv file and the groups tsv file are present`.

Reply:
Fixed in next commit.